keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -0,0 +1,248 @@
+---
+title: go modules
+authors:
+  - "@liggitt"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-api-machinery
+  - sig-release
+  - sig-testing
+reviewers:
+  - "@sttts"
+  - "@lavalamp"
+approvers:
+  - "@smarterclayton"
+  - "@thockin"
```

Comment:
I can be a reviewer or approver for this as needed.

Reply:
thanks, added as reviewer